Source/WebCore/platform/mac/UserActivityMac.mm

```diff
@@ -26,25 +26,30 @@
 #import "config.h"
 #import "UserActivity.h"
 
+#import "Logging.h"
+
 namespace WebCore {
 
 #if HAVE(NS_ACTIVITY)
 
-UserActivity::Impl::Impl(const char* description)
-    : m_description([NSString stringWithUTF8String:description])
+UserActivity::Impl::Impl(ASCIILiteral descriptionLiteral)
+    : m_description([NSString stringWithUTF8String:descriptionLiteral])
+    , m_descriptionLiteral(descriptionLiteral)
 {
 }
 
 void UserActivity::Impl::beginActivity()
 {
     if (!m_activity) {
+        RELEASE_LOG(ActivityState, "Began activity: %" PUBLIC_LOG_STRING, m_descriptionLiteral.characters());
```

Comment:
We're not going to have any context in the logging. "Began activity" here is too generic. What kind of activity.
This is why we tend to log the function name, e.g. : `"%p - UserActivity::Impl::beginActivity: description=%" PUBLIC_LOG_STRING`

Reply:
Also, is it really worth keeping the ASCIILiteral as a data member just to avoid calling `[m_description UTF8String]`?